tests/python/pants_test/pantsd/test_process_manager.py

```diff
@@ -63,19 +75,6 @@ def test_readwrite_metadata_by_name(self):
                 self.pmm.read_metadata_by_name(self.TEST_KEY, int), self.TEST_VALUE_INT
             )
 
-    @pytest.mark.skip(reason="flaky: https://github.com/pantsbuild/pants/issues/6836")
-    @pytest.mark.no_error_if_skipped
-    def test_deadline_until(self):
-        with self.assertRaises(ProcessManager.Timeout):
-            with self.captured_logging(logging.INFO) as captured:
```

Comment:
`captured_logging` no longer exists, and to use the standard caplog fixture we need the test to be a pytest function and not a unittest method.  So this has been broken out of the class.

A future change will do the same for the other tests in this file.